plugins/mac-avcapture/data/locale/en-US.ini

```diff
@@ -15,3 +15,9 @@ Auto="Auto"
 Unknown="Unknown (%1)"
 EnableAudio="Enable audio if supported by device"
 Resolution="Resolution"
+PortraitEffectWarning="Portrait system effect is active on the selected device"
+CenterStageWarning="Center Stage system effect is active on the selected device"
+StudioLightWarning="Studio Light system effect is active on the selected device"
+ReactionsWarning="Reaction effects are active for the selected device"
+BackgroundReplacementWarning="Background replacement is active for the selected device"
+MultipleEffectsWarning="Multiple system effects are active on the selected device"
```

Comment:
Personally I'd prefer keys like `Warning.PortraitEffect` etc for better readability. Additionally, some contain "Effect" and others don't, is that intentional?

Reply:
Agree; adjusted the naming and the descriptions to be more uniform.